src/usr/local/buildpack/tools/v2/yarn.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+function check_tool_requirements () {
+  check_command node
+  check_semver "$TOOL_VERSION"
+
+  if [[ ! "${MAJOR}" || ! "${MINOR}" || ! "${PATCH}" ]]; then
+    echo Invalid version: "${TOOL_VERSION}"
+    exit 1
+  fi
```

Comment:
can we create a helper to reduce duplicate code?

Reply:
will do